src/ray/common/task/scheduling_resources.h

```diff
@@ -147,6 +147,23 @@ class ResourceSet {
   /// \return Void.
   void AddResources(const ResourceSet &other);
 
+  /// \brief Aggregate resources from the other set into this set, adding any missing
+  /// resource labels to this set. The resource id will change to bundle_id + "_" +
```

Comment:
Can this be `placement_group_id + bundle_index + resource_id`? We can also add `placement_group_id + resource_id` to enable placing without bundle restrictions

Reply:
Yep, now the bundle_id is placement_group_id + index, so the name is placement_group_id + bundle_index + resource_id now.